src/sentry/static/sentry/app/components/events/interfaces/threads/getThreadStacktrace.ts

```diff
@@ -0,0 +1,30 @@
+import {Thread} from 'app/types/events';
```

Comment:
I think we are using .tsx extension everywhere, even if the file does not contain jsx.